server/item/stack.go

```diff
@@ -50,11 +50,18 @@ func (s Stack) WithArmourTrim(trim ArmourTrim) Stack {
 	if _, ok := s.item.(Armour); !ok {
 		return s
 	}
-
 	s.armourTrim = &trim
 	return s
 }
 
+// WithoutArmourTrim returns the current stack but with the passed ArmourTrim removed.
+func (s Stack) WithoutArmourTrim(trim ArmourTrim) Stack {
```

Comment:
Not sure it makes sense to require a trim being passed here, given that an item is only ever able to have one trim.

Reply:
that's true ill remove the trim.